label_studio/tests/README.md

```diff
@@ -0,0 +1,43 @@
+
+## Test tutorial
+
+
+### structure
+
+- functional tests
+    --pytest
+- smoke tests
+    --pytest write "actions" for simple real actions (endpoints, and core)
+- e2e tests
+    --pytesy parametrize fixture
+    --combination of actions and data
+
+
+e2e_actions - contain action functions
+
+e2e_test.ACTIONS - mapping 'action_name': action_function
+
+e2e_test.TestCase - runner for suits
```

Comment:
suits -> suites
same for suites.py

Reply:
? plural of suit is suits
there is a question about naming - 
case | suit | scenario
it makes sense to choose one for  file and class namings
scenarios.py + class Scenarios
or suits.py + class Suits
or cases.py + class Cases